.travis.yml

```diff
@@ -56,12 +62,23 @@ addons:
     - libatlas-base-dev
 
 before_install:
+  - SECONDS=0 && function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
No. This line is well-placed to be executed within a few seconds of the travis clock starting. If it is removed, then the bash clock will not be synchronized to the travis clock